packages/server-web/package.json

```diff
@@ -33,9 +33,10 @@
     "@babel/plugin-syntax-dynamic-import": "^7.0.0",
     "@babel/plugin-transform-runtime": "^7.1.0",
     "@babel/preset-env": "^7.1.0",
-    "@storybook/addon-options": "^3.4.11",
-    "@storybook/addon-storyshots": "^4.0.0-alpha.20",
-    "@storybook/vue": "^4.0.0-alpha.20",
+    "@storybook/addon-actions": "^4.0.0-alpha.23",
```

Comment:
do we really want to use an alpha version? i'm ok with it as it's just for testing... but if we don't need any of the new features, i'd still rather go for something more stable